http_queue_service.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"bytes"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strings"
+	"time"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+type httpClient interface {
+	Do(req *http.Request) (resp *http.Response, err error)
+}
+
+type httpQueue struct {
+	httpClient httpClient
+	endpoint   string
+}
+
+func newHttpQueueService(httpClient httpClient, endpoint string) httpQueue {
```

Comment:
You're right, but I wanted to display that a) it's using the queue interface and b) it's different from the http_service that's already in the repository.